telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -258,6 +259,8 @@ class TelnyxClient(
      */
     fun getSocketResponse(): LiveData<SocketResponse<ReceivedMessageBody>> = socketResponseLiveData
 
+    fun getWsMessageResponse(): LiveData<SocketResponse<WsMessageData>> = wsMessagesResponseLiveDate
```

Comment:
Can we add a method description here? You can use the method above as a reference